.github/workflows/deploy.yml

```diff
@@ -0,0 +1,344 @@
+# Check documentation at https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+name: Deploy
+
+on:
+  pull_request: ~
+  push:
+    branches:
+      - master
+
+# In order to use this GitHub Actions file, please define the following secrets:
+#  - GKE_PROJECT: project id of the Google Kubernetes Engine
+#  - GKE_SA_KEY: Service Account Key of the Google Kubernetes Engine
+#  - BLACKFIRE_SERVER_ID: Blackfire Server id
+#  - BLACKFIRE_SERVER_TOKEN: Blackfire Server token
+#  - CF_API_TOKEN: Cloudflare API token
+env:
+  COMPOSE_INTERACTIVE_NO_CLI: 1
+  GKE_CLUSTER: api-platform-demo
+  GCE_ZONE: europe-west1-c
+  CONTAINER_REGISTRY_BASE: eu.gcr.io/${{ secrets.GKE_PROJECT }}
+  DOMAIN: api-platform.com
+  SUBDOMAIN: demo
+  DEFAULT_BRANCH: master
+
+jobs:
+  deploy-api:
+    name: API
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2-beta
+
+      # Generate the branch name (refs/heads/feature/branch-1 => feature-branch-1)
+      - name: Inject slug/short variables
+        uses: rlespinasse/github-slug-action@v2.x
+
+      # Setup gcloud CLI
+      - uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          service_account_key: ${{ secrets.GKE_SA_KEY }}
+          project_id: ${{ secrets.GKE_PROJECT }}
+
+      # Configure Docker to use the gcloud command-line tool as a credential helper for authentication
+      - run: gcloud --quiet auth configure-docker
+
+      # Get the GKE credentials so we can deploy to the cluster
+      - run: gcloud container clusters get-credentials ${{ env.GKE_CLUSTER }} --zone ${{ env.GCE_ZONE }}
+
+      - name: Install Helm 3
```

Comment:
Helm is preinstalled in GitHub Actions' containers (https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md), this may not be necessary.